source/common/buffer/zero_copy_input_stream_impl.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+#include "envoy/buffer/buffer.h"
+
+#include "google/protobuf/io/zero_copy_stream.h"
+
+namespace Envoy {
+
+namespace Buffer {
+
+class ZeroCopyInputStreamImpl : public virtual google::protobuf::io::ZeroCopyInputStream {
```

Comment:
Yes, `TranscoderInputStreamImpl` inherits this class and `TranscoderInputStream` from transcoding library, which inherits `ZeroCopyInputStream`.